config/passport.js

```diff
@@ -1,7 +1,5 @@
 const passport = require("passport");
 const LocalStrategy = require("passport-local").Strategy;
```

Comment:
change LocalStategy to camelCase

Reply:
`LocalStrategy` is actually a class. 
If you look downwards it was instantiated when `passport.use` function was called